01-syntax/03-data-structures/the-sum-of-ranges/range.js

```diff
@@ -3,9 +3,29 @@
  *
  */
 /* global range */
-
-// → Your code here.
-
+function range(start, end, step) {
+  var i = 0;
```

Comment:
You may go with `var i = start || 0;` ....

Just take a look at https://addyosmani.com/blog/exploring-javascripts-logical-or-operator/